gh-pages/src/assets/css/style.css

```
@@ -0,0 +1,365 @@
+      format("svg");
+  font-display: swap;
+}
+body {
+  background-color: #fff;
+  padding: 3.5em;
+  font:
+    14px/1.5 "Noto Sans",
+    "Helvetica Neue",
+    Helvetica,
+    Arial,
+    sans-serif;
+  color: #727272;
+  font-weight: 400;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: #222;
+  margin: 0 0 1.25rem;
+}
+
+p,
+ul,
+ol,
+table,
+pre,
+dl {
+  margin: 0 0 1.25rem;
+}
+
+ul,
+ol {
+  padding-inline: 2em 0;
+}
+
+h1,
+h2,
+h3 {
+  line-height: 1.1;
+}
+
+h1 {
+  font-size: 2em;
+}
+
+h2 {
+  color: #393939;
+}
+
+h3,
+h4,
+h5,
+h6 {
+  color: #494949;
+}
+
+a {
+  color: #267cb9;
+  text-decoration: none;
+}
+
+a:hover,
+a:focus {
+  color: #069;
+}
+
+a small {
+  font-size: smaller;
+  color: #777;
+  margin-top: -0.3em;
+  display: block;
+}
+
+a:hover small {
+  color: #777;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-columns: 19.25em auto;
+  gap: 0 6.5em;
+  width: 61.5em;
+  margin: 0 auto;
+}
+
+blockquote {
+  border-left: 1px solid #e5e5e5;
+  margin: 0;
+  padding: 0 0 0 1.5em;
+  font-style: italic;
+}
+
+code,
+pre {
+  font-family:
+    Monaco,
+    Bitstream Vera Sans Mono,
+    Lucida Console,
+    Terminal,
+    Consolas,
+    Liberation Mono,
+    DejaVu Sans Mono,
+    Courier New,
+    monospace;
+  color: #333;
+}
+
+pre {
+  padding: 0.8em 1.1em;
+  background: #f8f8f8;
+  border-radius: 5px;
+  border: 1px solid #e5e5e5;
+  overflow-x: auto;
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+th,
+td {
+  text-align: left;
+  padding: 0.33em 0.66em;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+dt {
+  color: #444;
+  font-weight: 700;
+}
+
+th {
+  color: #444;
+}
+
+img {
+  max-width: 100%;
+}
+
+kbd {
+  background-color: #fafbfc;
+  border: 1px solid #c6cbd1;
+  border-bottom-color: #959da5;
+  border-radius: 3px;
+  box-shadow: inset 0 -1px 0 #959da5;
+  color: #444d56;
+  display: inline-block;
+  font-size: smaller;
+  line-height: 1;
+  padding: 3px 5px;
+  vertical-align: middle;
+}
+
+header {
+  width: 19.25em;
+  position: fixed;
+  grid-column: 1;
+  -webkit-font-smoothing: subpixel-antialiased;
+}
+
+header ul.nav {
+  padding-inline: 1em;
+}
+
+header ul.nav a[href] {
+  display: block;
+}
+
+header ul.nav a[href^="http"]::after {
+  content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12' viewBox='0 0 12 12'%3E%3Cpath fill='%2336c' d='M6 1h5v5L8.86 3.85 4.7 8 4 7.3l4.15-4.16zM2 3h2v1H2v6h6V8h1v2a1 1 0 0 1-1 1H2a1 1 0 0 1-1-1V4a1 1 0 0 1 1-1'/%3E%3C/svg%3E");
+  padding-inline-start: 0.1em;
+}
+
+strong {
+  color: #222;
+  font-weight: 700;
+}
+
+section {
+  grid-column: 2;
+  padding-block: 1.33em 4em;
+}
+
+small {
+  font-size: 11px;
+}
+
+hr {
+  border: 0;
+  background: #e5e5e5;
+  height: 1px;
+  margin: 0 0 20px;
+}
+
+footer {
+  width: 19.25em;
+  position: fixed;
+  bottom: 3.5em;
+  -webkit-font-smoothing: subpixel-antialiased;
+}
+
+@media print, screen and (max-width: 960px) {
```

No need to specifically list `print, screen`. No media types means `all`.

```suggestion
@media (max-width: 960px) {
```